src/controllers/itemdetailpage.js

```diff
@@ -647,7 +647,7 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "cardBuild
     function setTitle(item, apiClient) {
         var url = logoImageUrl(item, apiClient, {});
 
-        if (url = null) {
+        if (url == null) {
```

Comment:
LOL

Reply:
Since 11994657af29ad8f30eafe93b82f5ad66b12a0a8
But I think that `!=` (or just `!url`) should be used here, because `url` is used in the block.

Reply:
yes, I think it should be `url != null`

Reply:
Good catch! That is definitely the case.

Reply:
Fixed :sweat_smile: